pages.ja/common/llvm-ar.md

```diff
@@ -0,0 +1,7 @@
+# llvm-ar
+
+> このコマンドは `ar` のエイリアスです.
```

Comment:
Should this use this `dot`?
```suggestion
> このコマンドは `ar` のエイリアスです。
```

Reply:
Yes, the Japanese pages use the full-width period. But this should be fixed in [alias-pages.md](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/alias-pages.md#ja) first.